### ISSUE TITLE
Angular 4 update: Behaviour of nsRouterLink has been changed slightly

### ISSUE THREAD

**Issue by JohannesHoppe** [NativeScript/nativescript-angular#724]
Please take a look at this simple routing example:

```
const routes: Routes = [
  {
    path: 'books',
    component: BookListComponent
  },
  {
    path: 'books/:isbn',
    component: BookDetailsComponent
  }
];
```

With the normal Angular 4 router I can navigate from `books` to `books/XXX` via this link:
```
<a [routerLink]="theNumber"></a>
```
(full example [here](https://github.com/book-monkey2-build/iteration-3-http/blob/master/src/app/book-list/book-list.component.html#L5), demo [here](https://iteration-3-http.angular-buch.com/books), tested on both Angular2/Router3 and Angular4/Router4)

Before upgrading this worked fine for nativescript-angular, too:
```
<StackLayout [nsRouterLink]="theNumber"></StackLayout>
```
(full example [here](https://github.com/angular-buch/book-monkey-nativescript/blob/dddb5b7e6d782722cebb379c77615b7ab44870e5/app/book-list/book-list.component.html#L10))

But after upgrading to `nativescript-angular@1.5.0` the router cannot match any routes. I had to change the link like this:
```
<StackLayout [nsRouterLink]="['books', theNumber]"></StackLayout>
```
(full example [here](https://github.com/angular-buch/book-monkey-nativescript/blob/cd62d1f09000f0d1f4aa23890ba6f8221c0542fd/app/book-list/book-list.component.html#L10))

This difference between `nsRouterLink` and the normal `routerLink` is not desired , isn't it?!

**Comment by JohannesHoppe** [comment]
Great, I have seen `relativeTo` in the [code](https://github.com/NativeScript/nativescript-angular/commit/71058f85b9ba595d4ba506f2d09ef7351290fed3#diff-6c389d1ed296a82f0fce501df883d9e4R90). 

Do you have an ETA for the patch on NPM?

**Comment by sis0k0** [comment]
If everything is fine, we'll release 1.5.1 tomorrow.

**Comment by sis0k0** [comment]
Fixed with 1.5.1 patch.

**Comment by JohannesHoppe** [comment]
I can confirm this. Thanks for the super-fast fix! 👍
